src/api/games.js

```diff
@@ -61,4 +63,22 @@ games.delete('/:gameId', validator.params(defaultGameParams), async (req, res) =
   return status.success(res, { message: `Deleted game ${gameId}` });
 });
 
+games.get('/:gameId/players', validator.params(getGamePlayersParams), async (req, res) => {
+  const { params: { gameId } } = req;
+
+  const players = await gamesData.getPlayers(gameId);
+
+  return status.success(res, { ...players });
+});
+
+games.patch('/:gameId/players',
+  validator.params(defaultGameParams),
+  validator.body(patchPlayersBody), async (req, res) => {
+    const { params: { gameId }, body: { playerId } } = req;
+
+    const addedPlayer = await gamesData.addPlayer(gameId, playerId);
+
+    return status.success(res, { ...addedPlayer });
+  });
```

Comment:
I'm slightly unsettled by this being out a tab, but I see why it is. Maybe put the closing parenthesis on its own line? Otherwise, I feel like stuff past this point is still in scope of the route handler.
```suggestion
  }
);
```

Reply:
I was torn on which way to break convention. I'll switch it up so you can feel settled 😄